pre-workshop/pre-workshop-ja.md

```diff
@@ -642,34 +638,34 @@ If you have any questions about your rights or treatment as a research participa
 - ザンビア
 - ジンバブエ
 
-### *Demographic Questions* Thank you very much for participating in this survey.  We appreciate your input, which will help 
-us to create a diverse, inclusive and welcoming event.  The Carpentries seek to promote a scientific computing community that is an inclusive and welcoming environment for all. In order to further this goal, we hope to 
-understand the makeup of the community and the experiences of individuals within it. The data from this survey will help us to assess the diversity of our community.
+### *人口動態調査* このアンケート調査へのご協力ありがとうございます。あなたの回答は、多様的で包括性がある歓迎的なイベントの実施に 
+役立てます。カーペントリーズは全ての人に包括的で歓迎的な環境の科学的なコンピューティングコミュニティを推進しています。このゴールを達成するために、コミュニティの構成、1人1人のそこでの経験を 
+理解したいと望んでいます。このアンケートの回答は私たちのコミュニティの評価に役立ちます。
 
-### To which gender do you most identify? 
+### 性別を教えてください。 
 
-- Female 
-- Gender variant/non-conforming 
-- Male 
-- Prefer not to say 
+- 女性 
+- 性別バリアント / 不適合 
```

Comment:
This is important, and we should prioritize a culturally appropriate translation over a literal one. Here is what [yahoo uses](https://about.yahoo.co.jp/info/blog/20180613/diversity.html).
```suggestion
- その他
```